dashboard/api.py

```diff
@@ -367,3 +380,127 @@ def format_courserun_for_dashboard(course_run, status_for_user, certificate=None
         # TODO: here goes the logic to pull the current grade  # pylint: disable=fixme
         pass
     return formatted_run
+
+
+def get_student_enrollments(user, edx_client):
```

Comment:
you are probably right about this, but I think we spent too much time on this PR already.
I am willing to revisit the code if I have to touch again or if we need to cache something else coming from edX